.github/workflows/tests.yml

```diff
@@ -25,6 +26,9 @@ jobs:
         run: |
           pip config --site set global.progress_bar off
           make installdeps-complete
+      - if: ${{ matrix.featuretools_version == 'Main' }}
+        name: Install latest version of Featuretools from main branch
+        run: pip install git+https://github.com/alteryx/featuretools.git@main#egg=featuretools --upgrade
```

Comment:
I think pip is skipping installing `main` if the version numbers match. 

```
make installdeps-test
pip install git+https://github.com/alteryx/featuretools.git@main#egg=featuretools --upgrade
python -c "from featuretools.tests.entityset_tests.test_dask_es import test_single_table_dask_entityset"
```
runs cleanly while

```
pip install --upgrade pip
pip install git+https://github.com/alteryx/featuretools.git@main#egg=featuretools --upgrade
pip install pytest
python -c "from featuretools.tests.entityset_tests.test_dask_es import test_single_table_dask_entityset"
```
gives 

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'test_single_table_dask_entityset' from 'featuretools.tests.entityset_tests.test_dask_es'
```
Which makes sense, because that test was moved out of that file since the last release of featuretools

Reply:
Good catch I'll use the force reinstall option